ledger/participant-integration-api/src/test/lib/scala/platform/store/dao/JdbcLedgerDaoBackendH2DatabaseAppendOnly.scala

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.platform.store.dao
+
+import com.daml.platform.store.DbType
+import org.scalatest.AsyncTestSuite
+
+private[dao] trait JdbcLedgerDaoBackendH2DatabaseAppendOnly extends JdbcLedgerDaoBackend {
```

Comment:
This actually applies both to `JdbcLedgerDaoBackendH2DatabaseAppendOnly` and `JdbcLedgerDaoBackendH2Database`. Why not drop the word `Database`. It is implied and makes already a long indentifier even longer

Reply:
Good point, the `Postgres` variants do not use the "Database" suffix.

Reply:
sure thing